pypeit/spectrographs/keck_deimos.py

```diff
@@ -269,6 +269,7 @@ def check_headers(self, headers):
                              '6.CCDNAME': '14-4-3',
                              '7.CCDNAME': '14-4-2',
                              '8.CCDNAME': '14-5-2' }
+        headers[0]['INSTRUME'] = headers[0]['INSTRUME'][:6] # To handle both 'fresh' and KOA data
```

Comment:
DEIMOS data fresh off the telescope (as opposed to KOA data) has a different instrument name in the header, `DEIMOS: real science mosaic CCD subsystem with PowerPC in VME crate`. This seems like a relatively inoffensive way to handle it for now, but I'm sure there's a more elegant approach...